tests/jupyter/test_pythreejs.py

```diff
@@ -202,7 +202,7 @@ def test_export_after_show():
     pl = pyvista.Plotter(shape=(2, 2))
 
     # deleting rather than showing to save time
```

Comment:
Comment seems stale now.

Reply:
Fixed in 132132a6053b3bfbf7e2045115a31ffedd593bf8.